github/client.go

```diff
@@ -574,15 +604,11 @@ func (client *Client) newOctokitClient(auth octokit.AuthMethod) *octokit.Client
 	return c
 }
 
-func (client *Client) absolute(endpoint string) *url.URL {
-	u, _ := url.Parse(endpoint)
-	if u.Scheme == "" && client.Host != nil {
+func (client *Client) absolute(host string) *url.URL {
+	u, _ := url.Parse("https://" + host)
```

Comment:
Previously `absolute()` created a URL that looks like `https://example.com`, but its Host was blank, instead its Path property was `/example.com`. So I decided to fix it. We were only passing plain hostnames to `endpoint` before anyway, so I changed the argument and tests to reflect that.